static/builder.js

```diff
@@ -1493,16 +1545,33 @@ function displaySearchResultsAsync(items, start, div) {
 
 function getItemEnhancementLink(item) {
     var html = "";
+    
     if (weaponList.includes(item.type)) {
         html += '<div class="enchantment"><img src="img/dwarf.png" onclick="event.stopPropagation();selectEnchantedItem(\'' + item.id + '\')">';
```

Comment:
Probably remove the onclick here in EXCLUDE mode, no ?

Reply:
Maybe remove the functionality entirely when in Exclude mode?

Reply:
Yes, even better